Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -84,3 +85,85 @@ Deploy application
     cd "BlueprintBaseName.1/src/BlueprintBaseName.1"
     dotnet lambda deploy-serverless
 ```
+
+## Lambda Annotations
+This template now uses [Lambda Annotations](https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations/README.md) to bridge the gap between the Lambda programming model and a more idiomatic .NET model.
```

Comment:
Don't say `now`. In a year from now it will look dated and we won't remember to remove it.

Reply:
Removed